src/platformsh.js

```diff
@@ -1,67 +1,447 @@
-let jsonConfig;
-
-try{
-  jsonConfig = require('/run/config.json');
-} catch(err) {
-  // Mocking data in dev mode
-  jsonConfig = {
-    info: {
-      limits: {
-        cpu: 1
-      }
-    }
-  };
-}
+'use strict';
+
 /**
-* Read number of CPUs from environment or fallback to the _private_ configuration property
-* Useful for determining the number of processes to fork.
-*/
-function num_of_cpus() {
-  try {
-    if(process.env['OMP_NUM_THREADS']) {
-      return process.env['OMP_NUM_THREADS'];
-    }
-
-    return Math.ceil(jsonConfig.info.limits.cpu);
-  } catch (err) {
-    throw new Error('Could not get number of cpus');
-  }
+ * Decodes a Platform.sh environment variable.
+ *
+ * @param {string} value
+ *   Base64-encoded JSON (the content of an environment variable).
+ *
+ * @return {any}
+ *   An associative array (if representing a JSON object), or a scalar type.
+ */
+let decode = (value) => {
```

Comment:
You can use `function decode(value) {}`. Or at least `const` if you're feeling hipster.